node_agent/main.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 Istio Authors
```

Comment:
Aslo, regarding your comment on "You can avoid having "na" package inside node_agent dir, which is non-ideal". I am not sure why that is so. You can see the kubelet implementation that follows the same pattern as I did : https://github.com/kubernetes/kubernetes/tree/master/cmd/kubelet

Thanks,
Sarvani